manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MEGALimited.MEGAcmd
+PackageVersion: 1.5.0.0
+PackageLocale: en-US
+Publisher: MEGA Limited

```

```suggestion
Publisher: Mega Limited
```
This must match Control Panel.